f5-bigip-mixin/dashboards/f5-cluster-overview.libsonnet

```diff
@@ -0,0 +1,633 @@
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  options: {
+    displayMode: 'gradient',
+    minVizHeight: 10,
+    minVizWidth: 0,
+    orientation: 'horizontal',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    showUnfilled: true,
+    valueMode: 'color',
+  },
+  pluginVersion: '10.2.0-60139',
+};
+
+local topOutboundTrafficNodesPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'topk($k, increase(bigip_node_serverside_bytes_out{job=~"$job", instance=~"$instance"}[$__interval:]))',
+      datasource=promDatasource,
+      legendFormat='{{node}} - {{partition}} - {{instance}}',
+      format='time_series',
+      interval='1m',
+    ),
+  ],
+  type: 'bargauge',
+  title: 'Top outbound traffic nodes / $__interval',
+  description: 'Nodes with the highest outbound traffic.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      min: 0,
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'decbytes',
+    },
+    overrides: [],
+  },
+  options: {
+    displayMode: 'gradient',
+    minVizHeight: 10,
+    minVizWidth: 0,
+    orientation: 'horizontal',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    showUnfilled: true,
+    valueMode: 'color',
+  },
+  pluginVersion: '10.2.0-60139',
+};
+
+local topActiveMembersInPoolsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'topk($k, bigip_pool_active_member_cnt{job=~"$job", instance=~"$instance"})',
+      datasource=promDatasource,
+      legendFormat='{{pool}} - {{partition}} - {{instance}}',
+      format='time_series',
+    ),
+  ],
+  type: 'bargauge',
+  title: 'Top active members in pools',
+  description: 'Pools with the highest number of active members.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'thresholds',
+      },
+      mappings: [],
+      min: 0,
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'none',
+    },
+    overrides: [],
+  },
+  options: {
+    displayMode: 'gradient',
+    minVizHeight: 10,
+    minVizWidth: 0,
+    orientation: 'horizontal',
+    reduceOptions: {
+      calcs: [
+        'lastNotNull',
+      ],
+      fields: '',
+      values: false,
+    },
+    showUnfilled: true,
+    valueMode: 'color',
+  },
+  pluginVersion: '10.2.0-60139',
+};
+
+local topRequestedPoolsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'topk($k, increase(bigip_pool_tot_requests{job=~"$job", instance=~"$instance"}[$__interval:]))',
```

Comment:
increase means you have to change the title right? watch for this throughout

Reply:
updated